feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewViewModel.kt

```diff
@@ -235,6 +237,45 @@ class PreviewViewModel @Inject constructor(
         }
     }
 
+    fun captureImage(
+        contentResolver: ContentResolver,
+        imageCaptureUri: Uri?,
+        onImageCapture: (ImageCaptureEvent) -> Unit
+    ) {
+        Log.d(TAG, "captureImageWithUri")
+        viewModelScope.launch {
+            traceAsync(IMAGE_CAPTURE_TRACE, 0) {
```

Comment:
nit: You should either add a benchmark to `ImageCaptureLatencyBenchmark` or create a TODO to add one.